tensorflow/core/kernels/dml_data_format_dim_map.cc

```diff
@@ -71,34 +109,52 @@ class DmlDataFormaDimMapKernel : public DmlKernel {
     // gather operation.
     uint32_t src_dst_mapping_packed = 0;
     uint32_t left_shift = 0;
+    uint8_t additional_src_dst_mapping = 0;
```

Comment:
Suggestion: Although this works, conceptually it would be clearer to me (took a moment to figure out 🤨) to make the `src_dst_mapping_packed` a uint64_t (which simplifies the loop below), and then just `&` or `>>` to access the respective parts of it below, on lines 144 and 154. 🤷‍♀️